keyboards/mmd/km75/keyboard.json

```diff
@@ -0,0 +1,250 @@
+            {"matrix": [2, 11], "x": 175, "y": 6, "flags": 4},
+            {"matrix": [2, 10], "x": 160, "y": 6, "flags": 4},
+            {"matrix": [2, 9], "x": 145, "y": 6, "flags": 4},
+            {"matrix": [2, 8], "x": 129, "y": 6, "flags": 4},
+            {"matrix": [2, 7], "x": 114, "y": 6, "flags": 4},
+            {"matrix": [2, 6], "x": 99, "y": 6, "flags": 4},
+            {"matrix": [2, 5], "x": 84, "y": 6, "flags": 4},
+            {"matrix": [2, 4], "x": 69, "y": 6, "flags": 4},
+            {"matrix": [2, 3], "x": 53, "y": 6, "flags": 4},
+            {"matrix": [2, 2], "x": 38, "y": 6, "flags": 4},
+            {"matrix": [2, 1], "x": 23, "y": 6, "flags": 4},
+            {"matrix": [2, 0], "x": 4, "y": 6, "flags": 4},
+            {"matrix": [1, 0], "x": 0, "y": 6, "flags": 4},
+            {"matrix": [1, 1], "x": 11, "y": 6, "flags": 4},
+            {"matrix": [1, 2], "x": 26, "y": 6, "flags": 4},
+            {"matrix": [1, 3], "x": 41, "y": 6, "flags": 4},
+            {"matrix": [1, 4], "x": 56, "y": 6, "flags": 4},
+            {"matrix": [1, 5], "x": 72, "y": 6, "flags": 4},
+            {"matrix": [1, 6], "x": 87, "y": 6, "flags": 4},
+            {"matrix": [1, 7], "x": 102, "y": 6, "flags": 4},
+            {"matrix": [1, 8], "x": 117, "y": 6, "flags": 4},
+            {"matrix": [1, 9], "x": 133, "y": 6, "flags": 4},
+            {"matrix": [1, 10], "x": 148, "y": 6, "flags": 4},
+            {"matrix": [1, 11], "x": 163, "y": 6, "flags": 4},
+            {"matrix": [1, 12], "x": 178, "y": 6, "flags": 4},
+            {"matrix": [1, 13], "x": 201, "y": 6, "flags": 4},
+            {"matrix": [1, 14], "x": 224, "y": 6, "flags": 4},
+            {"matrix": [0, 13], "x": 213, "y": 6, "flags": 4},
+            {"matrix": [0, 12], "x": 194, "y": 6, "flags": 4},
+            {"matrix": [0, 11], "x": 179, "y": 6, "flags": 4},
+            {"matrix": [0, 10], "x": 164, "y": 6, "flags": 4},
+            {"matrix": [0, 9], "x": 149, "y": 6, "flags": 4},
+            {"matrix": [0, 8], "x": 129, "y": 6, "flags": 4},
+            {"matrix": [0, 7], "x": 114, "y": 6, "flags": 4},
+            {"matrix": [0, 6], "x": 99, "y": 6, "flags": 4},
+            {"matrix": [0, 5], "x": 84, "y": 6, "flags": 4},
+            {"matrix": [0, 4], "x": 65, "y": 6, "flags": 4},
+            {"matrix": [0, 3], "x": 50, "y": 6, "flags": 4},
+            {"matrix": [0, 2], "x": 34, "y": 6, "flags": 4},
+            {"matrix": [0, 1], "x": 19, "y": 6, "flags": 4},
+            {"matrix": [0, 0], "x": 0, "y": 6, "flags": 4}
+        ]
+    },
+    "url": "www.i-chip.cn",
+    "usb": {
+        "device_version": "0.0.1",
+        "force_nkro": true,
+        "pid": "0x3021",
+        "vid": "0x28E9"
+    },
+    "ws2812": {
+        "driver": "pwm",
+        "pin": "A6"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"matrix": [0, 0], "x": 0, "y": 0},
```

Comment:
Are the Tab, Caps Lock and Left Shift keys meant to be 1u?
![image](https://github.com/user-attachments/assets/55b49c06-b579-4d36-9db4-4b9400a0d032)

Reply:
updated :)